rust/src/dns/detect.rs

```diff
@@ -1,4 +1,4 @@
-/* Copyright (C) 2019 Open Information Security Foundation
+/* Copyright (C) 2024 Open Information Security Foundation
```

Comment:
When updating copyright years for existing files, we keep the initial year, like so, in this case:
2019-2024